ionic/components/modal/modal.ts

```diff
@@ -127,19 +131,49 @@ export class Modal extends ViewController {
 
 }
 
+@Component({
+  selector: 'ion-modal',
+  template: `
+    <div class="backdrop"></div>
+    <div class="modal-wrapper">
+      <div #wrapper></div>
+    </div>
+  `
+})
+class ModalComponent {
+
+  @ViewChild('wrapper', {read: ViewContainerRef}) wrapper: ViewContainerRef;
+
+  constructor(protected dynamicComponentLoader: DynamicComponentLoader, protected navParams: NavParams, protected viewController: ViewController) {
```

Comment:
Do we need to use `protected` instead of `private` here? Also, to stay consistent let's use `_loader`, `_navParams` and `_viewCtrl`